core.go

```diff
@@ -745,6 +745,36 @@ func ExtractChannel(src Mat, dst Mat, coi int) {
 	C.Mat_ExtractChannel(src.p, dst.p, C.int(coi))
 }
 
+// FindNonZero returns the list of locations of non-zero pixels
+//
+// For further details, please see:
+// https://docs.opencv.org/master/d2/de8/group__core__array.html#gaed7df59a3539b4cc0fe5c9c8d7586190
+//
+func FindNonZero(src Mat, idx Mat) {
+	C.Mat_FindNonZero(src.p, idx.p)
+}
+
+// Flip flips a 2D array around vertical, horizontal, or both axes.
+//
+// FlipCode	a flag to specify how to flip the array; 0 means flipping around the x-axis and positive value (for example, 1) means flipping around y-axis.
```

Comment:
How does that render in godoc?

Reply:
what exactly ? a colon behind the param name ?

Reply:
the `\t` between the argument name and its description